crates/ruff_cli/tests/integration_test.rs

```diff
@@ -1276,8 +1276,8 @@ fn diff_only_unsafe_fixes_available() {
         ])
         .pass_stdin("x = {'a': 1, 'a': 1}\nprint(('foo'))\n"),
         @r###"
-    success: false
-    exit_code: 1
+    success: true
+    exit_code: 0
```

Comment:
I think this is correct, and was wrong before.

Reply:
Agreed